src/pybind/mgr/dashboard/frontend/src/app/core/navigation/administration/administration.component.html

```diff
@@ -1,4 +1,5 @@
-<cds-overflow-menu [customTrigger]="customTrigger"
+<cds-overflow-menu [customTrigger]="customTrigger" 
+                   (click) ="closeModal()"
```

Comment:
this feels more like a hack or workaround doing it on per component wise. Ideally something like this should exist in about component itself and it should be smart enough to go away when you click on somewhere else or prevent you from clicking on other things. 

It could be the side-effect of the older bootstrap modal that we were using. Can you try to introduce carbon modal to the about component and see if the issue is gone?